hub/api/dataset.py

```diff
@@ -89,43 +81,50 @@ def __getitem__(self, item: Union[str, int, slice, Index]):
         else:
             raise InvalidKeyTypeError(item)
 
-    def __setitem__(self, item: Union[slice, str], value):
-        if isinstance(item, str):
-            tensor_key = item
+    def create_tensor(
+        self, name: str, chunk_size: int = DEFAULT_CHUNK_SIZE, dtype: str = "float64"
```

Comment:
Added `htype` since "type" is a python built-in (even though it's not restricted, it messes up syntax highlighting and intellisense).

Didn't write any presets for it, but there's a place to add them when we decide what the values should be.